src/devices/Lsm9Ds1/Lsm9Ds1.AccelerometerAndGyroscope.cs

```diff
@@ -0,0 +1,138 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.IO;
+using System.Device.Gpio;
+using System.Device.I2c;
+using System.Diagnostics;
+using System.Runtime.CompilerServices;
+using System.Device.I2c.Drivers;
+using System.Buffers.Binary;
+using System.Numerics;
+
+namespace Iot.Device.Lsm9Ds1
+{
+    public class Lsm9Ds1AccelerometerAndGyroscope : IDisposable
+    {
+        private const byte ReadMask = 0x80;
+        private I2cDevice _i2c;
+        private AccelerationScale _accelerometerScale;
+        private AngularRateScale _angularRateScale;
+
```

Comment:
I think devices like SPI and I2C should provide information such as I2cAddress, SpiClockFrequency in class. If I am a user, I would probably not look at examples or datasheet. For example, if the I2C device has multiple addresses, it can be provided by enum type.😁

Reply:
code like this:
```C#
// SPI
public const int SpiClockFrequency = 5000000;
public const SpiMode SpiMode = System.Device.Spi.SpiMode.Mode3;

// I2C
public const byte I2cAddress = 0x68;
public enum I2cAddress
{

}
```

Reply:
I couldn't find any I2C address in the datasheet except for the SenseHat where this sensor is - for sense HAT I've added that in the SenseHat version of the device